templates/storyboards/swift3.stencil

```diff
@@ -24,89 +20,74 @@ protocol StoryboardSceneType {
 }
 
 extension StoryboardSceneType {
-  static func storyboard() -> UIStoryboard {
-    return UIStoryboard(name: self.storyboardName, bundle: Bundle(for: BundleToken.self))
-  }
-
-  static func initialViewController() -> UIViewController {
-    guard let vc = storyboard().instantiateInitialViewController() else {
-      fatalError("Failed to instantiate initialViewController for \(self.storyboardName)")
-    }
-    return vc
+  static func storyboard() -> {{prefix}}Storyboard {
+    return {{prefix}}Storyboard(name: self.storyboardName, bundle: Bundle(for: BundleToken.self))
   }
 }
 
 extension StoryboardSceneType where Self: RawRepresentable, Self.RawValue == String {
```

Comment:
Yeah good idea (except that these should be `static let` not `static var` 😛 ), for the sam reason we removed enums `case` in the other templates. We shouldn't use `case` if it doesn't make sense to `switch` over them, and should prefer `static let` in those conditions indeed 👍